x-pack/solutions/security/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -586,14 +586,13 @@ const usePolicyTemplateInitialName = ({
 
     const currentIntegrationName = getMaxPackageName(integration, packagePolicyListByIntegration);
 
-    if (newPolicy.name === currentIntegrationName) {
-      return;
-    }
-
-    updatePolicy({
-      ...newPolicy,
-      name: currentIntegrationName,
-    });
+    updatePolicy(
+      {
+        ...newPolicy,
+        name: currentIntegrationName,
+      },
+      Array.isArray(packagePolicyListByIntegration)
```

Comment:
i suggest to put this value into a named constant so it will be easier to understand why this check is being done. in this use case maybe something like `hasPackagePoliciesLoaded `

also, note that an empty array is will still return true for `Array.isArray`, is this case something we care about? if so, probably good to additionally check for length.